pkg/rpc/context.go

```diff
@@ -135,6 +135,13 @@ func NewServer(ctx *Context) *grpc.Server {
 		)
 		opts = append(opts, grpc.UnaryInterceptor(interceptor))
 	}
+	// By default, gRPC disconnects clients that send "too many" pings,
+	// so we need to ensure that our server permits as many pings as our
+	// client is configured to send.
+	opts = append(opts, grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
```

Comment:
Since this is always included, I'd prefer to just put it in the initial definition of `opts` above.